### PR TITLE
Refactor language selection

### PR DESCRIPTION
**Pull request by talvasconcelos**
Refactored language selection with a loop.

Languages are defined in base.html as an array.

Possibly we should add an ISO code, for languages ex: `en_US, pt_PT, pt_BR` and an optional display also `display: 'EN'` , `display: 'PT'`, `display: PT(BR)`

### CODE REVIEW

**Review by dni — APPROVED** [review]

**Comment by motorina0** [comment]
@dni do I remember correctly that there was (related) a PR that added flags next to the language?
Will that be in conflict with this one?

File: lnbits/templates/base.html [hunk 1]
```diff
@@ -325,9 +285,22 @@
...
+      window.langs = [
+        { value: 'en', label: 'English', display: 'EN' },
+        { value: 'de', label: 'Deutsch', display: 'DE' },
+        { value: 'es', label: 'Español', display: 'ES' },
+        { value: 'jp', label: '日本語', display: 'JP' },
+        { value: 'fr', label: 'Français', display: 'FR' },
+        { value: 'it', label: 'Italiano', display: 'IT' },
+        { value: 'pi', label: 'Pirate', display: 'PI' },
+        { value: 'nl', label: 'Nederlands', display: 'NL' },
+        { value: 'we', label: 'Welsh' },
```
Comment (motorina0) [inline lnbits/templates/base.html:301]: Why no `display` field for `Polish` and `Pirate`

**Review by motorina0 — APPROVED** [review]

File: lnbits/templates/base.html (hunk 1 above)
Comment (callebtc) [reply lnbits/templates/base.html:301]: `Welsh` isn't a `Pirate` language afaik, but I could be wrong ;)

Comment (talvasconcelos) [reply lnbits/templates/base.html:301]: @motorina0 , it's an optional field, i don't know, and copilot didn't step in, how to spell `Polish` in ... polish!

@callebtc ??

Comment (arbadacarbaYK) [reply lnbits/templates/base.html:301]: Polski

Comment (arcbtc) [reply lnbits/templates/base.html:301]: Welsh should be CY, yep, please change

**Pull request closed without merging**